README.md

```diff
@@ -16,8 +16,13 @@ Instructions are as follows. Clone this repository:
 ```bash
 git clone https://github.com/hackersatcambridge/hac-website.git
 ```
+Navigate to the hac-website directory that you've now cloned. Create a file named `.env` and give it the following contents:
+```
+DATA_DIR=/root/hac-website/Data
```

Comment:
Would it be better to set this config through our `docker-compose` file? This means less setup and people can continue to use the `.env` setup on their Macs because `DATA_DIR` will need to be different for container vs macOS.